src/experimental/riff-raff-yaml-file/index.ts

```diff
@@ -155,48 +139,38 @@ export class RiffRaffYamlFileExperimental {
   private validateStacksInApp(): void {
     type Found = "✅";
     type NotFound = "❌";
-    type AppValidation = Record<ClassName, Record<StackTag, Record<Region, Record<StageTag, Found | NotFound>>>>;
+    type AppValidation = Record<StackTag, Record<Region, Record<StageTag, Found | NotFound>>>;
 
-    const { allCdkStacks, allStackTags, allStageTags, allRegions } = this;
+    const { allStackTags, allStageTags, allRegions } = this;
 
-    const checks: AppValidation = allCdkStacks.reduce((accClassName, { constructor: { name } }) => {
+    const checks: AppValidation = allStackTags.reduce((accStackTag, stackTag) => {
       return {
-        ...accClassName,
-        [name]: allStackTags.reduce((accStackTag, stackTag) => {
+        ...accStackTag,
+        [stackTag]: allRegions.reduce((accRegion, region) => {
```

Comment:
I think this is one for a future PR, but IIUC this validation is still too strict, as it looks [like using different regions for different `deployment`s is valid](https://public.riffraff.gutools.co.uk/docs/reference/riff-raff.yaml.md) from Riff-Raff's perspective?

Reply:
This use-case is catered for, I think. The [consent-management-platform](https://github.com/guardian/consent-management-platform/blob/fed48ed0703b1ad9be7ec240c2212fe6ddb13359/riff-raff.yaml) repo is one example of deploying to multiple regions, and https://github.com/guardian/consent-management-platform/pull/659 demonstrates compatibility. Does [this test](https://github.com/guardian/cdk/blob/0709c54afcb6c79600292143f2dd717c46f84b63/src/experimental/riff-raff-yaml-file/index.test.ts#L105-L163) describe the use-case you describe? Or would it be worth adding a test for multiple stacks and multiple regions?